src/configuration.rs

```diff
@@ -15,6 +15,15 @@ pub struct Config {
     services: ServicesList,
     hash: std::string::String,
     version: u32,
+    proxy_rules: Vec<MappingRules>,
+}
+
+#[derive(Default, Debug, Clone)]
+pub struct MappingRules {
+    pattern: std::string::String,
+    http_method: std::string::String, // @TODO this should be a enum, maybe from hyper
```

Comment:
There is a `Method` type in the `http` crate meant for this purpose, but we can do that later.

Reply:
yeah, let's do later.